get-started/installation-procedure.md

```diff
@@ -27,41 +32,39 @@ Fiddler Everywhere utilizes built-in .NET Core (cross-platform version .NET fram
     https://amazonaws.com/
 ```
 
->tip Fiddler Everywhere version 1.1.0 and above is supporting predefined OS proxy settings (for direct internet access). If you setup is accessing the internet via a predefined proxy, make sure you have the latest version of Fiddler Everywhere.
+>tip Fiddler Everywhere version 1.1.0 and above supports predefined OS proxy settings (for direct internet access). If your system is accessing the internet via a predefined proxy, make sure you have the latest version of Fiddler Everywhere.
 
 ## Install Fiddler Everywhere
- 
-Steps to install Fiddler Everywhere client on your system (macOS, Windows, or Linux)
 
->tip By default, the standard installation will allow you to track only non-secure HTTP traffic. To enable tracking and decoding of HTTPS, you will need to install a trusted certificate, as shown in the [HTTPS]({%slug decrypt-https-traffic%}) article.
+>tip After the installation, Fiddler Everywhere allows you to track only non-secure HTTP traffic. To enable capturing and decoding of HTTPS traffic, you need to install a trusted certificate, as shown in the [HTTPS]({%slug decrypt-https-traffic%}) article.
 
 ### macOS
 
-1. The Fiddler Everywhere installer is available for download on the [Progress website](https://www.telerik.com/download/fiddler-everywhere).
+1. Navigate to the [Fiddler Everywhere](https://www.telerik.com/download/fiddler-everywhere) download page.
 
-2. Click **Download for OS X** to start downloading the installer
+2. Select **Download for OS X** to download the installer.
 
 3. Run the **Fiddler Everywhere** dmg file to start the installation process.
 
 ### Windows
 
-1. The Fiddler Everywhere installer is available for download on the [Progress website](https://www.telerik.com/download/fiddler-everywhere).
+1. Navigate to the [Fiddler Everywhere](https://www.telerik.com/download/fiddler-everywhere) download page.
 
-2. Click **Download for Windows** to start downloading the installer
+2. Select **Download for Windows** to download the installer.
```

Comment:
Whenever possible, try to use [select](https://testdocs.telerik.com/style-guide/click-tap-select) as a universal term